RELEASE.md

```diff
@@ -0,0 +1,44 @@
+# Team
+
+The release team is made up of the following:
+
+* Rotating release engineer from one of the development teams
+* SRE team supporting deployment and CI/CD
+* Test Engineers that focus on integration tests
+* Performance Engineers running benchmarking tests
+
+# Release Process
+
+This is the release process for node releases
+
+1. Release Squad Lead will work with rotating engineer and dev leads to determine where to cut the releases from ledger, network and node. These will be published via CHaP and will follow the defined process of the team for versioning, etc...
+2. These will be integrated up the stack to the node to produce a release branch
+3. Release Squad Lead will work with Test/Performance sub-teams to initiate a testing round. This may be a tagged release or may be tip of branch.
+4. A release candidate will be deployed to preview/preprod by SRE
```

Comment:
This probably implies the rc should be published somewhere too, for people to deploy and download it.

Reply:
that's all automated thankfully :)